### PR TITLE
Add Twigcs linter

### PR DESCRIPTION
**Pull request by Ciloe**
Signed-off-by: Jimmy ESCRICH <jescrich@ruedesvignerons.com>

This PR due to add linter expected in this issue : https://github.com/friendsoftwig/twigcs/issues/38

### CODE REVIEW

File: syntax_checkers/twig/twigcs.vim [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+let g:loaded_syntastic_twig_twigcs_checker = 1
+
+let s:save_cpo = &cpo
+set cpo&vim
+
+function! SyntaxCheckers_twig_twigcs_GetLocList() dict
+    let makeprg = self.makeprgBuild({ 'args_after': '--reporter=emacs' })
+    " example report line : path/to/file/a.html.twig:1:8: warning - Unused variable "foo".
+    " available report values : ignore, info, warning, error
+    let errorformat = '%f:%l:%c: %t - %m'
```
Comment (lcd047) [inline syntax_checkers/twig/twigcs.vim:25]: `%t` in `errorformat` matches a single character, not something like `warning`.

Comment (Ciloe) [reply syntax_checkers/twig/twigcs.vim:25]: I can't see a documentation about the error format parameters. What can I use to match all report values possibilities ?

Comment (lcd047) [reply syntax_checkers/twig/twigcs.vim:25]: `:h errorformat`

File: plugin/syntastic/registry.vim [hunk 2]
```diff
@@ -98,7 +98,7 @@ let s:_DEFAULT_CHECKERS = {
         \ 'text':          [],
         \ 'trig':          ['rapper'],
         \ 'turtle':        ['rapper'],
-        \ 'twig':          ['twiglint'],
```
Comment (lcd047) [inline plugin/syntastic/registry.vim:101]: This changes the defaults for no reason.  Please don't.

File: syntax_checkers/twig/twigcs.vim (hunk 1 above)
Comment (Ciloe) [reply syntax_checkers/twig/twigcs.vim]: Like in phpCS I think I can `%f:%l:%c: %t%*[a-z] - %m`

**Comment by lcd047** [comment]
Sorry but you're making a mess out of this, you shouldn't patch things you don't understand.  I tried to fix it for you: commit 63741646.

**Pull request closed without merging**